docker/Dockerfile

```diff
@@ -26,14 +26,16 @@
 
 
 # --- Build the binary
-FROM golang:1.9.2-alpine3.7 AS build
+FROM golang:1.10.3-alpine3.7 AS build
```

Comment:
Can we bump to Go 1.11 now that it's out?

Reply:
looks like `golang:1.11.0-alpine3.8` is now available

Reply:
done